src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,42 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int [] principaldiagonal = new int [matrix.length];
```

Comment:
```suggestion
        int[] principalDiagonal = new int[matrix.length];
```
see the [guide](https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names) to make correct naming

Reply:
done

Reply:
done